modules/statics/src/base.ts

```diff
@@ -292,6 +292,10 @@ export enum CoinFeature {
    * This coin is onboarded on etheruem rollup chain
    */
   ETH_ROLLUP_CHAIN = 'eth-rollup-chain',
+  /**
+   * This coin is hidden
+   */
+  HIDDEN = 'hidden',
```

Comment:
Name is very ambiguous, when and where is it hidden?

Reply:
feature to be used from UI to hide a coin/token

Reply:
I'd agree and say name should be more explicit, or at least in the comment you can add `This coin is hidden from the UI`

Reply:
As discussed with @dpkjnr, we ll go with the existing DEPRECATED coin feature to hide gteth from gas tank; 
steps:
1. create an alpha version with deprecated feature for gteth
2. use this version to fix tests in wp
3. attach evidences from (2) here and merge this PR
4. create a PR for wp with test fixes